src/Form/Button.php

```
@@ -0,0 +1,112 @@
+     *
+     * @param string|UnitEnum|null $value Button type (`button`, `reset`, `submit`), or `null` to remove the attribute.
+     *
+     * @throws InvalidArgumentException if the provided value is not valid.
+     *
+     * @return static New instance with the updated `type` attribute.
+     *
+     * {@see ButtonType} for predefined enum values.
+     */
+    public function type(string|UnitEnum|null $value): static
+    {
+        Validator::oneOf($value, ButtonType::cases(), 'type');
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\UIAwesome\Html\Helper\Validator' in method 'type'. (undefined)

(StaticAccess)